wavelink/tracks.py

```diff
@@ -194,6 +194,8 @@ async def search(
             tracks = await node.get_tracks(cls, f"{cls._search_type}:{query}")
 
         if return_first:
+            if not tracks:
+                return None
```

Comment:
I think I kinda prefer it to be just a check on its own before the if return first statement